### PR TITLE
Hunting lodge Location

### PR DESCRIPTION
**Pull request by Maleclypse**
#### Summary

SUMMARY: Content "Hunting Lodge Location"

#### Purpose of change

Adds a hunting lodge location with a couple of variants.

#### Describe the solution

Adds overmap special, overmap terrain, maps and chunks for variants.

#### Describe alternatives you've considered

None

#### Testing
Add second floor, add drugs and dungeon chunks, add dungeon furniture, finalize itemgroups.

### CODE REVIEW

**Comment by Maleclypse** [comment]
@curstwist I'm not sure what I'm doing wrong but my special is repeating "lodge_ground1" instead of placing the "lodge_ground2".

File: data/json/mapgen/hunting_lodge.json [hunk 1]
```diff
@@ -0,0 +1,180 @@
+[
+  {
+    "type": "mapgen",
+    "method": "json",
+    "om_terrain": [ "lodge_ground1", "lodge_ground2" ],
```
Comment (curstwist) [inline data/json/mapgen/hunting_lodge.json:5]: "om_terrain": [ [ "lodge_ground1", "lodge_ground2" ] ],

double brackets should fix your problem (for all the merged maps).

File: data/json/mapgen/nested/lodge_nested.json [hunk 2]
```diff
@@ -0,0 +1,238 @@
...
+          { "item": "leather_shop_repair", "chance": 40, "repeat": [ 2, 4 ] }
+        ],
+        "d": [
+          { "item": "pants", "chance": 100 },
+          { "item": "shirts", "chance": 100 },
+          { "item": "leather_shop_accessories", "chance": 10 }
+        ],
+        "B": [
+          { "item": "SUS_tailoring_tool_drawer", "chance": 60, "repeat": [ 0, 1 ] },
+          { "item": "tailorbooks", "chance": 60, "repeat": [ 0, 1 ] },
```
Comment (Maleclypse) [inline data/json/mapgen/nested/lodge_nested.json:232]:
```suggestion
          { "item": "funnel", "chance": 60, "repeat": [ 0, 1 ] },
```

File: data/json/mapgen/nested/lodge_nested.json [hunk 3]
```diff
@@ -0,0 +1,238 @@
...
+          { "item": "SUS_tailoring_fasteners", "chance": 30, "repeat": [ 2, 6 ] }
```
Comment (Maleclypse) [inline data/json/mapgen/nested/lodge_nested.json:233]:
```suggestion
          { "item": "plastic_bag", "chance": 90, "repeat": [ 2, 6 ] }
```

File: data/json/mapgen/nested/lodge_nested.json [hunk 4]
```diff
@@ -0,0 +1,238 @@
...
+          { "item": "SUS_tailoring_fasteners", "chance": 30, "repeat": [ 1, 2 ] },
+          { "item": "leather_shop_repair", "chance": 40, "repeat": [ 2, 4 ] }
+        ],
+        "d": [
+          { "item": "pants", "chance": 100 },
+          { "item": "shirts", "chance": 100 },
+          { "item": "leather_shop_accessories", "chance": 10 }
+        ],
+        "B": [
+          { "item": "SUS_tailoring_tool_drawer", "chance": 60, "repeat": [ 0, 1 ] },
```
Comment (Maleclypse) [inline data/json/mapgen/nested/lodge_nested.json:231]:
```suggestion
          { "item": "coke", "chance": 60, "repeat": [ 13, 126 ] },
```

File: data/json/itemgroups/SUS/lodge.json [hunk 5]
```diff
@@ -0,0 +1,80 @@
...
+    "entries": [
+      { "group": "guns_shotgun_common", "prob": 30 },
+      { "group": "SUS_hunting_rifle", "prob": 20 },
+      { "group": "SUS_hunting_archery", "prob": 15 }
+    ]
+  },
+  {
+    "id": "cannibal_weapons",
+    "type": "item_group",
+    "//": "items found in a shotgun hunting space",
```
Comment (actual-nh) [inline data/json/itemgroups/SUS/lodge.json:47]: The comment - not quite right for `cannibal_weapons` - seems to be copied from the `hunting_lodge_weapons` item group above.

**Pull request merged**